languages/swift/exercises/concept/arrays/.meta/design.md

```diff
@@ -27,6 +27,5 @@ Of the primary Swift collection types, we chose to use the `Array` collection ty
 
 ## Prerequisites
 
-This exercise's prerequisites Concepts are:
-
 - `basics`: know how to work with `Int`s and how to assign and update variables and functions.
+- `conditionals`: know how to define an if statement, know how to define a guard statement.
```

Comment:
Could you also add this prerequisite it to the `config.json` file? Note that if an exercise has a prerequisite other than basics, you can omit the basics prerequisite. (see [the docs](https://github.com/exercism/v3/blob/master/docs/maintainers/determining-concepts.md#basics)).